db/eventlog.c

```diff
@@ -441,11 +442,12 @@ static void eventlog_add_newsql(const struct reqlogger *logger)
 
     cson_object_set(newobj, "time", cson_new_int(logger->startus));
     cson_object_set(newobj, "type",
-            cson_value_new_string("newsql", strlen("newsql")));
+            cson_value_new_string("newsql", sizeof("newsql") - 1));
```

Comment:
Let us optimize for readability. If you see https://godbolt.org/z/bGn9dK to convince yourself that compiler generates identical code for string literals.

Reply:
ok. will put back. Compiler Optimization makes sense but was unaware. thanks.